style.css

```diff
@@ -0,0 +1,105 @@
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto:wght@700&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap");
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+:root {
+  --color-primary-text-1: #ff6a01;
+  --color-supporting-text-4: #fef5e0;
+  --color-icons-8: #dbd8d7;
+  --color-description-text-11: #fff;
+  --color-background-1: #1c1a19;
+  --color-nav-bar: #3c3a39;
+}
+
+.header {
+  min-width: 375px;
+  background-color: var(--color-nav-bar, #3d3b39);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 2px 24px 2px 12px;
+}
+
+.header .logo {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 700px;
+  color: var(--color-icons-8);
+  text-decoration: none;
+  font-size: 18px;
+  line-height: 24px;
+  padding-left: 12px;
+}
+
+.hero {
+  width: 100%;
+  min-width: 375px;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 12px;
+  padding: 120px 24px;
+  background-image: url(./assets/images/illustration-header-mobile.png);
+  background-repeat: no-repeat;
+  background-position: bottom right;
+  background-color: var(--color-background-1);
+}
+
+.hero .greeting {
+  position: relative;
+  width: 300px;
+  font-family: 'Crete Round', serif;
+  color: var(--color-primary-text-1, #ff6b00);
+  font-size: 56px;
+  font-weight: 400px;
+  line-height: 64px;
+  letter-spacing: 0.37px;
+  font-style: normal;
+  text-align: left;
+  margin-bottom: 0;
+}
+
+.hero .title {
+  display: flex;
+  align-items: center;
+  flex-grow: 0;
+  width: 327px;
```

Comment:
- [ ] Kindly consider removing the fixed width from line 56 as suggested by the previous reviewer as it goes against the responsive design. usually, we try to give widths in percentages if necessary.